tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -588,12 +588,19 @@ func checkVmStorageCompliance(client clientset.Interface, storagePolicy string)
 // statefulset, deployment and volumes of statfulset created
 func createStsDeployment(ctx context.Context, client clientset.Interface, namespace string,
```

Comment:
If we are creating new util for Deployment and StatefulSet creation, let's try to keep it more generic

I am also writing the same kind of util for multi-vc automation

Pass arguments to this util like below 

StatefulSet policy - parallel/default
Pod NodeAffinity passing - Yes/No
replica count for sts/deployment
Do we need sts and deployment both - Yes/No
Access mode - 
Allowed Topology  - Yes/No

Like that, I can see a few you have covered, lets's sync up. I will show you what is a multi-vc requirement and then i can skip new util creation for it